apps/graphql/lib/graphql/schema/scaffold.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GraphQl.Schema.Scaffold do
+  use GraphQl.Schema.Base
+
+  alias GraphQl.Resolvers.Scaffolds
+
+  object :tfprovider do
```

Comment:
I think making this type named `terraform_provider` or something a bit less abbreviated might be ideal so that external devs wouldn't be confused